sirepo/template/template_common.py

```diff
@@ -9,6 +9,7 @@
 from pykern import pkjinja
 from pykern.pkcollections import PKDict
 from pykern.pkdebug import pkdc, pkdlog, pkdp, pkdexc
+from sirepo import simulation_db
```

Comment:
This import should be localized in analysis_job_dispatch. There are circular import issues here.